imageregistry/v1/tests/configs.imageregistry.operator.openshift.io/ChunkSizeMiB.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Config"
+crdName: configs.imageregistry.operator.openshift.io
+featureGate: ChunkSizeMiB
+tests:
+  onCreate:
+    - name: Should be able to create a minimal Config
+      initial: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          replicas: 1
+      expected: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+          replicas: 1
+    - name: Should be able to configure ChunkSizeMiB for storage s3
+      initial: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          replicas: 1
+          storage:
+            s3:
+              bucket: bucket-1
+              region: region-1
+              ChunkSizeMiB: 10
+      expected: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+          replicas: 1
+          storage:
+            s3:
+              bucket: bucket-1
+              region: region-1
+              ChunkSizeMiB: 10485760
+    - name: Should not be able to configure ChunkSizeMiB for storage s3 without details
+      initial: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          replicas: 1
+          storage:
+            s3:
+              bucket: bucket-2
+              region: region-2
+      expected: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+          replicas: 1
+          storage:
+            s3:
+              bucket: bucket-2
+              region: region-2
+    - name: Should reject configuration with value less than 5 in ChunkSizeMiB for storage s3
+      initial: |
+        apiVersion: imageregistry.operator.openshift.io/v1
+        kind: Config
+        spec:
+          replicas: 1
+          storage:
+            s3:
+              bucket: bucket-2
+              region: region-2
+              ChunkSizeMiB: 2
+      expectedError: "spec.storage.s3.chunksizemib in body should match '^(5\\d{0,2}|[1-4]\\d{3}|5120)$'"
```

Comment:
The casing on `chunksisemib` is not as expected, and the regex looks off, looks like the tests need a good update?